docs/standard/microservices-architecture/architect-microservice-container-applications/direct-client-to-microservice-communication-versus-the-API-Gateway-pattern.md

```diff
@@ -132,9 +131,9 @@ With Azure API Management, you can secure your APIs using a key, a token, and IP
 
 In this guide and the reference sample application (eShopOnContainers), the architecture is limited to a simpler and custom-made containerized architecture in order to focus on plain containers without using PaaS products like Azure API Management. But for large microservice-based applications that are deployed into Microsoft Azure, we encourage you to evaluate Azure API Management as the base for your API Gateways in production.
 
-**Ocelot.** For simpler approaches, a lightweight API Gateway like Ocelot is recommended. [Ocelot](https://github.com/ThreeMammals/Ocelot) is an open-source .NET Core-based API Gateway especially made for microservices architecture that need unified points of entry into their system. It's lightweight, fast, scalable and provides routing and authentication among many other features.
+For simpler approaches, a lightweight API Gateway like **[Ocelot](https://github.com/ThreeMammals/Ocelot)** is recommended. Ocelot is an Open Source .NET Core based API Gateway especially made for microservices architecture that need unified points of entry into their system. It's lightweight, fast, scalable and provides routing and authentication among many other features.
```

Comment:
changes to the second sentence shouldn't be done

Reply:
Created two sub-sections